### ISSUE TITLE
[Comment] The Windows Package Manager

### ISSUE THREAD

**Issue by RehanSaeed** [RehanSaeed/rehansaeed.github.io#496]
https://rehansaeed.com/windows-package-manager/

**Comment by WorldMaker** [comment]
@RehanSaeed Yeah, `winget search` doesn't seem to turn up any paid applications, but `winget list` does and `winget install --id 9nbhcs1lx4r0` almost works. It gives you the description of Paint.NET and asks you to agree to terms then I got a `Verifying/Requesting package acquisition failed: server error` after agreeing to the terms. It's possible the error is just that I already have it installed already, but it sounds like there's just no support at all for paid apps yet per microsoft/winget-cli#908 

Hopefully something they solve, because there are several other paid apps in the Microsoft Store I'd want to auto-install in a machine setup script, beyond of course Paint.NET.
